guides/doc-Managing_Hosts/topics/proc_registering-a-host-to-project-using-the-global-registration-template.adoc

```diff
@@ -0,0 +1,174 @@
+This method uses the global registration template that gives you complete control over the process of the host registration.
+
+ifeval::["{build}" != "satellite"]
+Note that you can extend the parameters by plug-ins. For more information, see https://github.com/theforeman/foreman/blob/develop/developer_docs/how_to_create_a_plugin.asciidoc.
+endif::[]
+
+.Prerequisites
+* The {Project} user that generates the `curl` command must have the `create_hosts` permission.
+* You must have root privileges on the host that you want to register.
+
+.Procedure
+. In {Project} web UI, navigate to *Hosts* > *Provisioning Templates* and locate and click *Linux registration default*.
+
+. Click the *Association* tab.
+
+. Select the operating systems that you want to register.
+
+. Navigate to *Hosts* > *Operating Systems* and click the operating system that you want to register.
+
+. Click *Submit*.
+
+. Click the *Templates* tab.
+
+. From the *Registration template* list, select *Linux registration default*.
+
+. Click *Submit*.
+
+. Navigate to *Hosts* > *All Hosts* > *Register Host*.
+
+. Optional: If you want to register hosts to {SmartProxy}, enter the following command on the {SmartProxy} to enable required modules:
++
+[options="nowrap", subs="+quotes,attributes"]
+----
+# foreman-installer --foreman-proxy-registration \
+--foreman-proxy-templates \
+--foreman-proxy-template-url 'http://{smartproxy-example-com}'
+----
+
+. What fields are required for Satellite and Foreman?
+
+. Optional: From the *Host Group* list, select the host group to associate the hosts with.
+
+. From the *Operating System* list, select the operating system of hosts that you want to register.
+
+. Optional: If you want to register hosts to a {SmartProxy}, from the *{SmartProxy}* list, select the {SmartProxy} to register hosts to.
+
+ifeval::["{build}" == "satellite"]
+. Optional: If you do not want to register hosts with Insights, from the *Insights* list, select *No*. What are the defaults for those two?
+endif::[]
+
+. Optional: If you do not want to deploy {Project} SSH keys to hosts, from the *Remote Execution* list, select *No*.
+
+. Optional: {Project} uses the JSON Web Tokens (JWT) for authentication.
+This duration of this token defines how long the generated `curl` command works.
+If you want to change the duration of the token, enter the required duration to the *Token lifetime (hours)* field.
+You can set the duration to any number of hours.
++
+Note that {Project} uses the permissions of the user who generated the `curl` command for authorization of hosts.
+If user loses or gains additional permissions, the permissions of the JWT change too.
+Therefore, do not delete, block, or change permissions of the user during the token duration.
+
+. Optional: What is remote execution interface?
+
+ifeval::["{build}" != "satellite"]
+. In the *Activation Key(s)* field, enter one or more activation keys to assign to registered hosts.
```

Comment:
```suggestion
. In the *Activation Key(s)* field, enter one or more activation keys to assign to hosts.
```
because they are not actually registered at this point

Reply:
You are right, thank you